core/src/main/java/com/orange/ouds/core/theme/OudsTypography.kt

```diff
@@ -49,7 +51,9 @@ data class OudsTypography(
     val labelStrongSmall: TextStyle,
 )
 
-
+/**
+ * @suppress
+ */
 fun OudsFontSemanticTokens.getTypography(fontFamily: FontFamily) = OudsTypography(
```

Comment:
This method could be `internal`. In that case we can remove the `@suppress` annotation.

Same comment for other methods of the same kind.